README.md

```diff
@@ -385,6 +384,23 @@ $client->close();
 
 
 ```
+#### Writing via UDP
+
+UDP Writer Requirements:
+1. Installed ext-sockets
+1. Since Influxdb 2.0+ does not support UDP protocol natively you need to install and configure Telegraf plugin: https://docs.influxdata.com/telegraf/v1.16/plugins/#socket_listener
+1. Extra config option passed to client: udpPort. Optionally you can specify udpHost, otherwise udpHost will parsed from url option
+ 
+```php
+$client = new InfluxDB2\Client(["url" => "http://localhost:8086", "token" => "my-token",
+    "bucket" => "my-bucket",
+    "org" => "my-org",
+    "precision" => InfluxDB2\Model\WritePrecision::NS,
+    "udpPort" => 8094,
+]);
+$writer = $client->createUdpWriter();
+$writer->write('h2o,location=west value=33i 15');
+ ```
```

Comment:
Please add clarification why you need this... something like https://github.com/influxdata/influxdb-client-php/pull/50#issuecomment-732154011 and also something like `$write->close()`should be in the example code.